openedx/core/djangoapps/content/block_structure/tasks.py

```diff
@@ -2,14 +2,22 @@
 Asynchronous tasks related to the Course Blocks sub-application.
 """
 import logging
+from capa.responsetypes import LoncapaProblemError
 from celery.task import task
 from django.conf import settings
+from edxval.api import ValInternalError
+from lxml.etree import XMLSyntaxError
 from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.exceptions import ItemNotFoundError
```

Comment:
These imports seem odd to me; what's the logic on the 3-blocks approach again? Language level imports, then third-party imports, then edx-platform imports?